it/test/helpers/ItTestBase.scala

```diff
@@ -64,10 +65,16 @@ trait ItTestBase extends PlaySpec with GuiceOneServerPerSuite {
     Enrolment("HMRC-CUS-ORG").withIdentifier("fake-identifier", Gen.alphaNumStr.sample.get)
   private val authResult              = Some("internalId") and Enrolments(Set(ourEnrolment))
 
-  def authorisedUser: OngoingStubbing[Future[Option[String] ~ Enrolments]] =
-    when(authConnector.authorise(any, eqTo(authFetch))(any, any)).thenReturn(
+  def authorisedUserWithAnswers: OngoingStubbing[Future[Option[String] ~ Enrolments]] = {
```

Comment:
I agree, Chris - I'll do that and say done when done. And Beth, this is just here since the tests will fail if the user has no answers (will redirect to the profile setup page, as per other logic implemented). The answers are empty and I think we can iterate as and when need comes along. If there we other cases it's just a matter of putting "authorisedUserWithSomeParticularAnswers" -> obviously not with that as the variable name though... perhaps with a simpler refactor, too.